blueocean-dashboard/src/main/js/components/Capability.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import update from 'react-addons-update';
```

Comment:
It is fairly lightweight It has no deps and is 150lines of code. Its a good library i would use again I think.